docs/content/en/docs/clustermgmt/observability/cluster-verify.md

```diff
@@ -8,40 +8,117 @@ date: 2017-01-05
-capi-system                         capi-controller-manager-59f4547955-2ks8t                         2/2     Running   0          47m
-capi-webhook-system                 capi-controller-manager-bb4dc9878-2j8mg                          2/2     Running   0          47m
-capi-webhook-system                 capi-kubeadm-bootstrap-controller-manager-6b4cb6f656-qfppd       2/2     Running   0          47m
-capi-webhook-system                 capi-kubeadm-control-plane-controller-manager-bf7878ffc-rgsm8    2/2     Running   0          47m
-capi-webhook-system                 capv-controller-manager-5668dbcd5-v5szb                          2/2     Running   0          47m
-capv-system                         capv-controller-manager-584886b7bd-f66hs                         2/2     Running   0          47m
 
+**Verify an EKS Anywhere cluster from the management cluster**
+
+First, set up `KUBECONFIG` environment variable for the management cluster:
+```
+export KUBECONFIG=<path-to-management-cluster-kubeconfig>
+```
+
+You may verify the control plane is up and running by filtering the pods by the `control-plane=controller-manager` label.
+```
+kubectl get pod -A -l control-plane=controller-manager
+NAMESPACE                           NAME                                                             READY   STATUS    RESTARTS      AGE
+capi-kubeadm-bootstrap-system       capi-kubeadm-bootstrap-controller-manager-8665b88c65-v982t       1/1     Running   0             3d21h
+capi-kubeadm-control-plane-system   capi-kubeadm-control-plane-controller-manager-67595c55d8-z7627   1/1     Running   0             3d21h
+capi-system                         capi-controller-manager-88bdd56b4-wnk66                          1/1     Running   0             3d21h
+capv-system                         capv-controller-manager-644d9864dc-hbrcz                         1/1     Running   1 (15h ago)   3d21h
+eksa-packages                       eks-anywhere-packages-784c6fc8b9-2t5nr                           1/1     Running   0             3d21h
+etcdadm-bootstrap-provider-system   etcdadm-bootstrap-provider-controller-manager-6bcdd4f5d7-wvqw8   1/1     Running   0             3d21h
+etcdadm-controller-system           etcdadm-controller-controller-manager-6f96f5d594-kqnfw           1/1     Running   0             3d21h
 ```
 
 You may also check the status of the cluster control plane resource directly. 
 This can be especially useful to verify clusters with multiple control plane nodes after an upgrade.
+
 ```
-kubectl get kubeadmcontrolplanes.controlplane.cluster.x-k8s.io
-NAME                       INITIALIZED   API SERVER AVAILABLE   VERSION              REPLICAS   READY   UPDATED   UNAVAILABLE
-supportbundletestcluster   true          true                   v1.20.7-eks-1-20-6   1          1       1
+kubectl get kubeadmcontrolplanes.controlplane.cluster.x-k8s.io -n eksa-system $CLUSTER_NAME
```

Comment:
nit: not too important but maybe add an instruction to export CLUSTER_NAME as well

Reply:
I was suppose to remove `$CLUSTER_NAME` here actually. The output shown gives all the kubeadmcontrolplanes 

Thanks